axis_linear: 1
axis_angular: 2
enable_button: 4  # L2 shoulder button
enable_turbo_button: 6  # L1 shoulder button
